ui/src/app/views/workspace-share/component.spec.ts

```diff
@@ -147,5 +147,58 @@ describe('WorkspaceShareComponent', () => {
         .toEqual(WorkspaceAccessLevel.OWNER);
   }));
 
+  it('validates and allows usernames', fakeAsync(() => {
+    spyOn(workspaceSharePage.fixture.componentRef.instance.workspacesService, 'shareWorkspace')
```

Comment:
I'd revert the visibility change. There are two ways you can get this without reaching into the component:
- Store a reference to `WorkspaceService` in the above `beforeEach()`
- `TestBed.get(WorkspaceService)`